parabeac.dart

```diff
@@ -71,33 +71,35 @@ ${parser.usage}
 
   _basePath = io.Directory.current.path;
 
-  var install = await Process.start('bash', [
-    '${Directory.current.path}/pb-scripts/install.sh',
-  ]);
+  if (_os != 'WIN') {
+    var install = await Process.start('bash', [
```

Comment:
@snwagbata I believe both `install.sh` and `merge-plugins.sh` should be run regardless of platform. `install.sh` can be omitted only if Figma is being run. Feel free to ask questions if you have any! Also, apologies for the delay in the PR review!

Reply:
The problem is that `.sh` files do not run on PowerShell and CMD on windows. I've had success getting them to run using git bash though

Reply:
I see. I think the problem is that if we don't merge plugins we will not be able to detect when people want eggs and if someone in windows wants to use Sketch they will not be able to run the Sketch Asset Converter. Do you happen to know if there is an alternative solution to running these scripts on Windows? I usually run parabeac-core on unix so I'm not sure what the best approach would be for this 🤔

Reply:
The alternative would be to use the Windows Subsystem for Linux. I attempted rewriting `install.sh` to a batch file a few days ago but it seems the Sketch Asset Converter has a dependency that only works on unix